### ISSUE TITLE
Add possibility to log trackables without logging the cache

### ISSUE THREAD

**Issue by okainov** [cgeo/cgeo#9504]
**Describe the solution you'd like**

Sometimes I either forget to log my trackables during logging the cache, or trackable logging just didn't for some time, so I have to log trackables again. Currently my plan is to log them with "write note" log to GC service and then remove the log from GC website, but it's quite inconvenient. It would be great if c:geo would allow just logging my trackables in some cache, but without sending log to the geocaching service 

**Additional context**
See #9496 as some kind of trigger for this FR

**Comment by fm-sys** [comment]
Isn't it possible via `search > tracking code`?

**Comment by SammysHP** [comment]
At least at GC you can't drop or visit a trackable without writing a log entry.

**Comment by Lineflyer** [comment]
For GK its possible:
- Search for trackable code (e.g. "GK10000")
- Create logs for this trackable
- Select Drop or Visit
- You can now enter the Geocode of the cache or coordinates for this log

For GC trackables AFAIK its not possible (or only via API?)

**Comment by okainov** [comment]
>Isn't it possible via search > tracking code

Not really. I mean, it is possible to log a single trackable, yeah. But it's almost useless.

From log cache view I can:
- Log all my trackables in one shot (you're not gonna log every single one separately if you have 10+, do you?)
- Log GKs without entering tracking code (as the website already knows it)
- Log without entering cache code (as from the cache it's already present in the backend)

And those are exactly the points.
